### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by gladjohn**
Fixes #
N/A

**Changes proposed in this request**
Update CHANGELOG.md for MSAL 4.47.0 release

**Testing**
n/a

**Performance impact**
none

**Documentation**
- [x] All relevant documentation is updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+4.47.0
+==========
+
+### New Features
+- The new MSAL logging feature is now generally available. `WithExperimentalFeatures()` is no longer required when calling `WithLogging()`. See [3548](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3548)
+
+### Bug Fixes
+- Check if scopes are passed when new preview broker is invoked. See [#3654](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3654)
+- Minor fixes to perf test project. See [#3662](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3662)
```
Comment (trwalke) [inline CHANGELOG.md:9]: Don't think we need to mention this as it is not relevant to the customer

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+4.47.0
+==========
+
+### New Features
+- The new MSAL logging feature is now generally available. `WithExperimentalFeatures()` is no longer required when calling `WithLogging()`. See [3548](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3548)
```
Comment (trwalke) [inline CHANGELOG.md:5]: Need to mention this https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2877

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+
+### Bug Fixes
+- Check if scopes are passed when new preview broker is invoked. See [#3654](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3654)
```
Comment (trwalke) [inline CHANGELOG.md:8]:
```suggestion
- MSAL will now throw an exception if no scopes are passed when the new preview broker is invoked. See [#3654](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3654)
```

**Comment by trwalke** [comment]
Also, add in these PRs when they are merged
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3677
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3678

File: CHANGELOG.md (hunk 2 above)
Comment (trwalke) [reply CHANGELOG.md:5]: I think you need this one too
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3496

I added them in the suggestion

```suggestion
- The new MSAL logging feature is now generally available. `WithExperimentalFeatures()` is no longer required when calling `WithLogging()`. See [3548](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3548)
- Adding IsProofOfPosessionSupportedByClient api to be used to determine if the current broker is able to support Proof-of-Posession. See [3496](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3496) 
- Adding ability to turn off the default retry-once policy on 5xx errors. See [2877](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2877)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+- Minor fixes to perf test project. See [#3662](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3662)
+- Update logic that calculates if exception is retry-able. See [#3661](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3661)
+- Add logging for Preview Broker `RemoveAccountAsync` API. See [#3658](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3658)
+- Adding Null account check on Preview Broker `RemoveAccountAsync` API. See [#3657](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3657)
+- Use correct authority in multi-cloud + WAM scenario. See [#3637](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3637)
```
Comment (pmaytak) [inline CHANGELOG.md:13]:
```suggestion
- `MsalServiceException.IsRetryable` is now correctly set. See [#3661](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3661)
- Added extra logging in Preview Broker `RemoveAccountAsync` API. See [#3658](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3658)
- Added a check for null account in Preview Broker `RemoveAccountAsync` API. See [#3657](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3657)
- `AuthenticationResult` now shows correct authority for multi-cloud requests using WAM. See [#3637](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3637)
```

**Review by pmaytak — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,17 @@
+4.47.0
+==========
+
+### New Features
+- The new MSAL logging feature is now generally available. `WithExperimentalFeatures()` is no longer required when calling `WithLogging()`. See [3548](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3548)
+
```
Comment (pmaytak) [inline CHANGELOG.md:6]:
```suggestion
- Adds new public builder API accepting instances of `ITelemetryClient`. See [3533](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3533).
- Added logic to log some acquire token data via the new telemetry pipeline. See [3534](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3534).

```

**Review by trwalke — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,20 @@
+4.47.0
+==========
+
+### New Features
+- The new MSAL logging feature is now generally available. `WithExperimentalFeatures()` is no longer required when calling `WithLogging()`. See [3548](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3548), [wiki](https://aka.ms/msal-net-logging).
+- Adding IsProofOfPosessionSupportedByClient api to be used to determine if the current broker is able to support Proof-of-Posession. See [3496](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3496) 
+- Adding ability to turn off the default retry-once policy on 5xx errors. See [2877](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2877)
+- Adds new public builder API accepting instances of `ITelemetryClient`. See [3533](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3533).
+- Added logic to log some acquire token data via the new telemetry pipeline. See [3534](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3534).
+
```
Comment (jmprieur) [inline CHANGELOG.md:10]: Don't you mention the support of MAUI?

Comment (gladjohn) [reply CHANGELOG.md:11]: added please review @jmprieur and @SameerK-MSFT

Comment (jmprieur) [reply CHANGELOG.md:11]: What about InTune, @gladjohn ?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,22 @@
+4.47.0
+==========
+
+### New Features
+- Support for .NET MAUI is now generally available for iOS, Windows and Android targets. The package also works with UWP. Refer to [`MauiStatus.md`](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/MauiStatus.md) for details.
+- The new MSAL logging feature is now generally available. `WithExperimentalFeatures()` is no longer required when calling `WithLogging()`. See [3548](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3548), [wiki](https://aka.ms/msal-net-logging).
+- Adding IsProofOfPosessionSupportedByClient api to be used to determine if the current broker is able to support Proof-of-Posession. See [3496](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3496) 
+- Adding ability to turn off the default retry-once policy on 5xx errors. See [2877](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2877)
+- Adds new public builder API accepting instances of `ITelemetryClient`. See [3533](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3533).
```
Comment (neha-bhargava) [inline CHANGELOG.md:9]: Nit: The telemetry pipeline work is Experimental for now to allow changes to API. Should we add in an Experimental section?

**Review by neha-bhargava — APPROVED** [review]

File: CHANGELOG.md (hunk 6 above)
Comment (gladjohn) [reply CHANGELOG.md:11]: Sameer says no new Intune in 4.47, already delivered in 4.46.2

**Pull request merged**